### PR TITLE
update document by feedback

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

SSIA

### Environment:

- Go Version: 1.18.3
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.7

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/contributing/coding-style.md [hunk 1]
```diff
@@ -252,37 +262,37 @@ c := &Something{
 }
 ```
 
-To initialize complex structs, we can use [functional option pattern](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis). Please read [server.go](https://github.com/vdaas/vald/blob/main/internal/servers/servers.go) and [option.go](https://github.com/vdaas/vald/blob/main/internal/servers/option.go) for the reference implementation.
-The options implementation should be separated as another file called `option.go` to improve the readability of the source code, and the method name should start with `With` word to differentiate with other methods.
+To initialize complex structs, we can use [a functional option pattern](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis). Please read [server.go](https://github.com/vdaas/vald/blob/main/internal/servers/servers.go) and [option.go](https://github.com/vdaas/vald/blob/main/internal/servers/option.go) for the reference implementation.
```
Comment (hlts2) [inline docs/contributing/coding-style.md:265]:
```suggestion
To initialize complex structs, we can use a [functional option pattern](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis). Please read [server.go](https://github.com/vdaas/vald/blob/main/internal/servers/servers.go) and [option.go](https://github.com/vdaas/vald/blob/main/internal/servers/option.go) for the reference implementation.
```

**Review by hlts2 — APPROVED** [review]
LGTM

File: docs/tutorial/vald-agent-standalone-on-docker.md [hunk 2]
```diff
@@ -153,27 +152,27 @@ This chapter will use NGT for the core engine of Vald Agent.
 
 1. Download dataset
 
-    In this tutorial. we use [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
+    In this tutorial. we use [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
```
[the remark this answers is not in the document]
Comment (vankichi) [reply docs/tutorial/vald-agent-standalone-on-docker.md:155]:
```suggestion
    In this tutorial, we use [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
```

File: docs/contributing/unit-test-guideline.md [hunk 3]
```diff
@@ -163,43 +165,41 @@ func AgeTest(val int32) (ok bool) {
...
-  1. When input is `...interface{}`
-      - we have to write all test cases which satisfies `...interface{}` as same as [basic test case](#Basic). For example, `val = 1`, `val = "input"`, `val = []float64{2020.12}` and so on.
-  1. When input is not `...interface{}` but `...[]int`, `...[]string` or etc
-      - we have to create only slice pattern test cases, which is the same as not create test cases with a single vale.
-      - we should test with boundary cases, for example, we should test with `val = []int{math.MaxInt64()}` when the input value is `...[]int`.
+Our basic test case depends on the type of two variadic arguments.
 
+1. When input is `...interface{}`
+   - We have to write all test cases which satisfy `...interface{}` as same as [a basic test case](#Basic). For example, `val = 1`, `val = "input"`, `val = []float64{2020.12}` and so on.
+1. When input is not `...interface{}` but `...[]int`, `...[]string` or etc
```
[the remark this answers is not in the document]
Comment (vankichi) [reply docs/contributing/unit-test-guideline.md:194]:
```suggestion
1. When input is not `...interface{}` but `...[]int`, `...[]string` or etc.
```

File: docs/contributing/unit-test-guideline.md [hunk 4]
```diff
@@ -163,43 +165,41 @@ func AgeTest(val int32) (ok bool) {
...
-
-  1. When input is `...interface{}`
-      - we have to write all test cases which satisfies `...interface{}` as same as [basic test case](#Basic). For example, `val = 1`, `val = "input"`, `val = []float64{2020.12}` and so on.
-  1. When input is not `...interface{}` but `...[]int`, `...[]string` or etc
-      - we have to create only slice pattern test cases, which is the same as not create test cases with a single vale.
-      - we should test with boundary cases, for example, we should test with `val = []int{math.MaxInt64()}` when the input value is `...[]int`.
+Our basic test case depends on the type of two variadic arguments.
 
+1. When input is `...interface{}`
+   - We have to write all test cases which satisfy `...interface{}` as same as [a basic test case](#Basic). For example, `val = 1`, `val = "input"`, `val = []float64{2020.12}` and so on.
```
Comment (hlts2) [inline docs/contributing/unit-test-guideline.md:193]:
```suggestion
   - We have to write all test cases which satisfy `...interface{}` as same as a [basic test case](#Basic). For example, `val = 1`, `val = "input"`, `val = []float64{2020.12}` and so on.
```

**Review by hlts2 — APPROVED** [review]
LGTM

File: docs/tutorial/get-started.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ Vald is a highly scalable distributed fast approximate nearest neighbor dense ve
 Vald is designed and implemented based on Cloud-Native architecture.
 
 This tutorial shows how to deploy and run the Vald components on your Kubernetes cluster.
-And, Fashion-mnist is used as an example of a dataset.
+And, fashion-MNIST is used as an example of a dataset.
```
Comment (datelier) [inline docs/tutorial/get-started.md:7]:
```suggestion
And, Fashion-MNIST is used as an example of a dataset.
```

File: docs/tutorial/get-started.md [hunk 6]
```diff
@@ -189,12 +189,12 @@ In this tutorial, you will deploy the basic configuration of Vald that is consis
 
 ## Run Example Code
 
-In this chapter, you will execute insert vectors, search vectors, and delete vectors to your Vald cluster using the example code.<br>
-The [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.
+In this chapter, you will execute insert, search, and delete vectors to your Vald cluster using the example code.<br>
+The [fashion-MNIST](https://github.com/zalandoresearch/fashion-MNIST) is used as a dataset for indexing and search query.
```
Comment (datelier) [inline docs/tutorial/get-started.md:193]:
```suggestion
The [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.
```

File: docs/tutorial/get-started.md [hunk 7]
```diff
@@ -206,15 +206,15 @@ If you are interested in, please refer to [SDKs](../user-guides/sdks.md).<br>
 
 1.  Download dataset
 
-    Download [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) that is used as a dataset for indexing and search query.
+    Download [fashion-MNIST](https://github.com/zalandoresearch/fashion-MNIST) that is used as a dataset for indexing and search query.
```
Comment (datelier) [inline docs/tutorial/get-started.md:209]:
```suggestion
    Download [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) that is used as a dataset for indexing and search query.
```

File: docs/tutorial/get-started.md [hunk 8]
```diff
@@ -206,15 +206,15 @@ If you are interested in, please refer to [SDKs](../user-guides/sdks.md).<br>
...
 
     Move to the working directory
 
     ```bash
     cd example/client
     ```
 
-    Download fashion-mnist testing dataset
+    Download fashion-MNIST testing dataset
```
Comment (datelier) [inline docs/tutorial/get-started.md:217]:
```suggestion
    Download Fashion-MNIST testing dataset
```

File: docs/tutorial/get-started.md [hunk 9]
```diff
@@ -223,7 +223,7 @@ If you are interested in, please refer to [SDKs](../user-guides/sdks.md).<br>
 1.  Run Example
 
     We use [`example/client/main.go`](https://github.com/vdaas/vald/blob/main/example/client/main.go) to run the example.<br>
-    This example will insert and index 400 vectors into the Vald from the fashion-mnist dataset via [gRPC](https://grpc.io/).
+    This example will insert and index 400 vectors into the Vald from the fashion-MNIST dataset via [gRPC](https://grpc.io/).
```
Comment (datelier) [inline docs/tutorial/get-started.md:226]:
```suggestion
    This example will insert and index 400 vectors into the Vald from the Fashion-MNIST dataset via [gRPC](https://grpc.io/).
```

File: docs/tutorial/get-started.md [hunk 10]
```diff
@@ -304,7 +304,7 @@ If you are interested in, please refer to [SDKs](../user-guides/sdks.md).<br>
 
     1.  load
 
-        - Loading from fashion-mnist dataset and set id for each vector that is loaded. This step will return the training dataset, test dataset, and ids list of ids when loading is completed with success.
+        - Loading from fashion-MNIST dataset and set id for each vector that is loaded. This step will return the training dataset, test dataset, and ids list of ids when loading is completed with success.
```
Comment (datelier) [inline docs/tutorial/get-started.md:307]:
```suggestion
        - Loading from Fashion-MNIST dataset and set id for each vector that is loaded. This step will return the training dataset, test dataset, and ids list of ids when loading is completed with success.
```

File: docs/tutorial/vald-agent-standalone-on-docker.md [hunk 11]
```diff
@@ -153,27 +152,27 @@ This chapter will use NGT for the core engine of Vald Agent.
...
+    This tutorial uses [Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
 
     ```bash
     # move to the working directory
     cd example/client/agent
     ```
 
     ```bash
-    # download fashion-mnist testing dataset
+    # download Fashion-mnist testing dataset
```
Comment (datelier) [inline docs/tutorial/vald-agent-standalone-on-docker.md:163]:
```suggestion
    # download Fashion-MNIST testing dataset
```

**Review by datelier — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]
LGTM
